platform/ios/resources/ru.lproj/Localizable.stringsdict

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>MAP_A11Y_VALUE</key>
+	<dict>
+		<key>NSStringLocalizedFormatKey</key>
+		<string>Zoom %dx
+%#@count@ visible</string>
+		<key>count</key>
+		<dict>
+			<key>NSStringFormatSpecTypeKey</key>
+			<string>NSStringPluralRuleType</string>
+			<key>NSStringFormatValueTypeKey</key>
+			<string>ld</string>
+			<key>one</key>
+			<string>%d аннотации(й)</string>
+			<key>few</key>
+			<string>%d аннотации(й)</string>
+			<key>many</key>
+			<string>%d аннотации(й)</string>
```

Comment:
@vkurchatkin, sorry to bother you again, but could you help translate the one/few/many forms of this message? Thanks!